packages/flame_tiled/lib/src/tiled_component.dart

```diff
@@ -57,7 +79,7 @@ class TiledComponent<T extends FlameGame> extends Component with HasGameRef<T> {
     );
   }
 
-  Vector2 _computeSize() {
+  static NotifyingVector2 _computeSize(RenderableTiledMap tileMap) {
```

Comment:
Does it need to be a `NotifyingVector2` here?

Reply:
You cannot change the type from position component .

Reply:
`PositionComponent` takes in a `Vector2` for super though, where this is the only place that you use this?

Reply:
Not implementing `PositionComponent` means we'd have to implement: 

```
        AnchorProvider,
        AngleProvider,
        PositionProvider,
        ScaleProvider,
        CoordinateTransform
```
And a lot of other logic from there. If PositonComponent only gave you a Vector2 - and had another way to register an observer on the vector, then maybe that's a work around?

Reply:
No no, I'm not suggesting you should not extend `PositionComponent`, I'm simply asking to change the return of this static method to `Vector2` since everything will still work like that. 😅
(Size will still be a `NofifyingVector2` in the end, but this method doesn't need to expose that)

Reply:
/me squints

I totally missed that context. Sorry! Ill see if I can do that at lunch time.